manifest_staging/deploy/gatekeeper.yaml

```diff
@@ -3276,6 +3276,55 @@ spec:
     served: true
     storage: true
 ---
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
To avoid user confusion we should not add syncsets to the manifest until we have a controller that can actuate them.